docs/literate/make.jl

```diff
@@ -0,0 +1,122 @@
+using Literate
```

Comment:
```suggestion
using Literate: Literate
```
Are all functions from Literate.jl always prefixed with `Literate.`? If so, this makes it more clear to people reading the code.